Modules/TOF/toftrending.json

```
@@ -52,7 +52,7 @@
           "userorcontrol"
         ],
         "updateTrigger": [
-          "newobject:ccdb:TOF/MO/TaskDigits/TOFRawsMulti"
+          "newobject:ccdb:TOF/MO/TaskDigits/Multiplicity/Integrated"
```

Since it is an object in QCDB, please use `qcdb`. Otherwise you are referring to the actual CCDB, which just by luck could be the same place.
```suggestion
          "newobject:qcdb:TOF/MO/TaskDigits/Multiplicity/Integrated"
```